### ISSUE TITLE
Crop Image after upload

### ISSUE THREAD

**Issue by MitchellQ** [ssm-deepcove/website#179]
This is a possibly a minor issue the crop image button in the media information page is inoperable, also it is visible in the information pages for audio and file types.

Do we want to allow users to crop the image after upload?

**Comment by samuelgrant** [comment]
This isn't something I am going to be work on for a while. @jacksos101 Do you think you could work on this and a couple of small issues next week?

**Comment by jacksos101** [comment]
I don't think we really need this feature, do we?

**Comment by samuelgrant** [comment]
No.

**Comment by MitchellQ** [comment]
So we should remove this button of it won't be used

**Comment by samuelgrant** [comment]
I can't imagine it would be too hard to implement, and it would be nice to have. At this point I think the bigger problem is that Sam and I are suffering from project faituge.

I sent him a message today with the last four things that need to get done, anything else would be a bonus, but a bonus I don't think will occur.

tldr; If @MitchellQ or @jacksos101 want to add the feature in a little bit, we keep the button. If neither of you want to make it work, we delete it.

**Comment by MitchellQ** [comment]
You could just hide it and if we want to work on it at some point its still there.

**Comment by samuelgrant** [comment]
@MitchellQ Please add the class d-none to the button
`className="d-none"` and then close this issue when done.

**Comment by MitchellQ** [comment]
Class has been added
